extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -19,9 +19,16 @@ import {listen} from '../../../src/event-helper';
 import {dev} from '../../../src/log';
 
 const TAG = 'amp-viewer-messaging';
-const sentinel_ = '__AMPHTML__';
-const requestSentinel_ = sentinel_ + 'REQUEST';
-const responseSentinel_ = sentinel_ + 'RESPONSE';
+const APP = '__AMPHTML__';
+
+/**
```

Comment:
Do we currently have any kind of whitelist of origins we are willing to talk to or are we going for everyone fully relying on trusted origin?
I'd prefer to start with a flexible whitelist (that people can add themselves to with a PR).